sdk/storagemover/Azure.ResourceManager.StorageMover/api/Azure.ResourceManager.StorageMover.netstandard2.0.cs

```diff
@@ -296,30 +296,61 @@ namespace Azure.ResourceManager.StorageMover.Models
         public static Azure.ResourceManager.StorageMover.Models.StorageMoverAgentPropertiesErrorDetails StorageMoverAgentPropertiesErrorDetails(string code = null, string message = null) { throw null; }
         public static Azure.ResourceManager.StorageMover.StorageMoverData StorageMoverData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, System.Collections.Generic.IDictionary<string, string> tags = null, Azure.Core.AzureLocation location = default(Azure.Core.AzureLocation), string description = null, Azure.ResourceManager.StorageMover.Models.StorageMoverProvisioningState? provisioningState = default(Azure.ResourceManager.StorageMover.Models.StorageMoverProvisioningState?)) { throw null; }
         public static Azure.ResourceManager.StorageMover.StorageMoverEndpointData StorageMoverEndpointData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, Azure.ResourceManager.StorageMover.Models.EndpointBaseProperties properties = null) { throw null; }
         public static Azure.ResourceManager.StorageMover.StorageMoverProjectData StorageMoverProjectData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, string description = null, Azure.ResourceManager.StorageMover.Models.StorageMoverProvisioningState? provisioningState = default(Azure.ResourceManager.StorageMover.Models.StorageMoverProvisioningState?)) { throw null; }
     }
+    public partial class AzureKeyVaultSmbCredentials : Azure.ResourceManager.StorageMover.Models.Credentials
+    {
+        public AzureKeyVaultSmbCredentials() { }
+        public string PasswordUri { get { throw null; } set { } }
```

Comment:
Change the type to URI

Reply:
Same as the other one. Could you help advise on how to clean up the passwordUri and usernameUri in this case? thanks!

Reply:
@yifanz7 See an example: https://github.com/Azure/azure-sdk-for-net/blob/047862626a1db94dfaee4cec64d8d87ced7fa600/sdk/appconfiguration/Azure.Data.AppConfiguration/src/autorest.md?plain=1#L20
For above case, I think the value should be `uri` to be aligned with Swagger spec: https://swagger.io/docs/specification/data-models/data-types/#string

Reply:
@archerzz This directive seems to aim at changing the swagger from string to uri, while in the swagger the type was changed to string by design since an empty string needs to be allowed for cleaning up the credential properties. 
Currently the issue is when the type is Uri, I wasn't able to find a way to input an empty string for PasswordUri and UsernameUri. Wondering if there's a workaround for this? So that we can input an empty string while keeping the type as Uri

Reply:
> @archerzz This directive seems to aim at changing the swagger from string to uri, while in the swagger the type was changed from Uri to string by design since an empty string needs to be allowed for cleaning up the credential properties. Currently the issue is when the type is Uri, I wasn't able to find a way to input an empty string for PasswordUri and UsernameUri. Wondering if there's a workaround for this? So that we can input an empty string while keeping the type as Uri

Got it. So basically, the empty string is used as a Patch delete, right?

Reply:
@archerzz Yes, input empty string to PasswordUri and UsernameUri to cleanup/delete the existing values